src/nucleicacid.jl

```diff
@@ -73,28 +71,18 @@ end
 # Print
 # -----
 
-function Base.show(io::IO, nt::DNA)
-    if isvalid(nt)
-        if nt == DNA_Gap
-            write(io, "DNA_Gap")
-        else
-            write(io, "DNA_", Char(nt))
-        end
-    else
-        write(io, "Invalid DNA")
-    end
-    return
-end
+showprefix(::Type{DNA}) = "DNA"
+showprefix(::Type{RNA}) = "RNA"
```

Comment:
Minor suggestion: `show` looks an I/O function to me while `showprefix` is not. I think `prefix` is enough.